build.gradle

```diff
@@ -29,7 +29,8 @@ allprojects {
     group = project.GROUP
 
     repositories {
-        jcenter()
         google()
+        jcenter()
+        maven { url 'https://jitpack.io' }
```

Comment:
Not sure if we want to commit this line. Won't it cause every branch to get pushed up by jitpack?

Reply:
Yeah, I think we might not need this as we only depend on the support library. The `google` maven repo should have all we need. Maybe we could switch back to `mavenCentral` instead of `jcenter` and avoid the issue @oshamahue mentioned altogether.

Reply:
ok I removed the jitpack repo. It is not really have to do with jitpack uploads. I added to check if the SNAPSHOT version is working good and forgot to remove. 

I think we should keep JCenter over maven central. It is now default repo on Android and it is bigger&faster over maven central.